packages/button-react/src/Button.tsx

```diff
@@ -4,23 +4,24 @@ import classNames from "classnames";
 interface Props {
     children: ReactNode;
     className?: string;
-    onClick: MouseEventHandler<HTMLButtonElement>;
+    onClick?: MouseEventHandler<HTMLButtonElement>;
     forceCompact?: boolean;
     inverted?: boolean;
+    type?: "button" | "reset" | "submit";
```

Comment:
Er dette kanskje et av tilfellene der vi bare bør ekstende hva man kan gjøre med et button-element?

Kanskje et issue at vi ikke har disabled state på våre knapper.

Reply:
Tenkte på det. Bakdelene er jo som du sier at vi ikke har disabled state, og at du får fem hundre forslag i IntelliSense som kanskje drukner våre "egne" props litt 😅 Men er nok godt mulig at det er verdt det likevel! Disabled state er det vel en viss sjanse for at vi blir nødt til å implementere selv om vi ikke i utgangspunktet har lyst...